client/packages/system/src/Patient/ListView/Toolbar.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import {
+  useTranslation,
+  AppBarContentPortal,
+  SearchBar,
+  FilterController,
+} from '@openmsupply-client/common';
+import { PatientRowFragment } from '../api';
+
+export const Toolbar: FC<{
+  filter: FilterController;
+}> = ({ filter }) => {
+  const t = useTranslation('patients');
+
+  const key = 'lastName' as keyof PatientRowFragment;
+  const filterString = (filter.filterBy?.[key]?.like as string) || '';
+
+  return (
+    <AppBarContentPortal
+      sx={{
+        paddingBottom: '16px',
+        flex: 1,
+        justifyContent: 'space-between',
+        display: 'flex',
+      }}
+    >
+      <SearchBar
```

Comment:
an unasked for offroad - thought that a filter here is helpful and rounds things out